tests/test_oauth2client.py

```diff
@@ -144,6 +146,23 @@ def get(self, key, default=None):
     return Info(self._headers)
 
 
+@contextlib.contextmanager
+def mock_module_import(module):
+  """Place a dummy object in sys.modules to mock an import test"""
+  parts = module.split('.')
+  for n in range(len(parts)):
+    path = '.'.join(parts[:n+1])
+    sys.modules[path] = object()
+  
+  try:
+    yield
+
+  finally:
+    for n in range(len(parts)):
```

Comment:
In the first half of the function create a sequence of the dummy entries added so that you don't have to recompute them to remove them here in the second half.